tests/drivers/build_all/opamp/prj.conf

```diff
@@ -0,0 +1,6 @@
+# Copyright 2025 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_TEST=y
+CONFIG_TEST_USERSPACE=y
```

Comment:
I am not sure if any user-space tests are needed. It was previously agreed the opamp API shall not provide any syscals and/or user-mode access, since opamps are always integral part of other drivers in context of a OS.